### PR TITLE
Fix for issue #227 dark theme flashing. 

### PR DESCRIPTION
**Pull request by rohit-nair**
Setting theme server side by writing and reading cookie. 
Also renamed 'default' theme as Light consistent with other implementations.
Removed localstorage implementation.

### CODE REVIEW

**Comment by G-Rath** [comment]
Yes please! Love this site, but the flash hurts my eyes big time 😬 

Any chance of getting this merged?

File: explainshell/web/static/js/es.js [hunk 1]
```diff
@@ -27,6 +29,48 @@ var assignedcolors = {};
 var vtimeout,
     changewait = 250;
 
+
+// From MDN's Library
```
Comment (stoivo) [inline explainshell/web/static/js/es.js]: Add a link

**Comment by stoivo** [comment]
@idank Looks good, I think you can merge.

**Comment by idank** [comment]
merged as https://github.com/idank/explainshell/commit/bcad627013972b442ed57e2846d831f6c239a44a thanks!

**Pull request closed without merging**